release.mk

```diff
@@ -120,15 +119,15 @@ minor-release:
 #
 .PHONY: patch-release
 patch-release:
-	@echo "INFO: Create feature branch and update the versions. Target branch $(RELEASE_BRANCH)"
-	$(MAKE) create-branch NAME=$(BRANCH_PATCH) BASE=$(RELEASE_BRANCH)
+	@echo "INFO: Create feature branch and update the versions. Target branch $(BASE_BRANCH)."
+	$(MAKE) create-branch NAME=$(BRANCH_PATCH) BASE=$(BASE_BRANCH)
 	$(MAKE) update-version VERSION=$(RELEASE_VERSION)
 	$(MAKE) update-docs VERSION=$(RELEASE_VERSION)
 	$(MAKE) update-version-makefile VERSION=$(PROJECT_MAJOR_VERSION)\.$(PROJECT_MINOR_VERSION)
-	$(MAKE) update-version-legacy VERSION=$(NEXT_RELEASE) PREVIOUS_VERSION=$(CURRENT_RELEASE)
-	$(MAKE) create-commit COMMIT_MESSAGE="docs: update docs versions to $(RELEASE_VERSION)"
+	$(MAKE) update-version-legacy VERSION=$(NEXT_RELEASE) PREVIOUS_VERSION=$(RELEASE_VERSION)
```

Comment:
`NEXT_RELEASE` uses the `RELEASE_VERSION` and increases the patch version.

`RELEASE_VERSION` is defined in the GitHub action, see https://github.com/elastic/apm-server/blob/963396c6b2ca588898d01c668b00793a5bd67d80/.github/workflows/run-patch-release.yml#L47